hack/test-cmd.sh

```diff
@@ -1654,6 +1654,51 @@ __EOF__
 
 
   ######################
+  # Lists              #
+  ######################
+
+  kube::log::status "Testing kubectl(${version}:lists)"
+
+  ### Create a List with objects from multiple versions
+  # Command
+  kubectl create -f - "${kube_flags[@]}" << __EOF__
+apiVersion: v1
```

Comment:
suggest adding a file of this config under `hack/testdata/` and then `create -f` it here